ExoDiPhotonAnalyzer/test/crab_cfg_template.py

```diff
@@ -33,6 +33,8 @@
 config.section_("Data")
 config.Data.inputDataset = dataset
 config.Data.inputDBS = 'global'
+config.Data.outLFNDirBase = '/store/user/ciperez/DiPhotonAnalysis/'
```

Comment:
Please don't add code that breaks the existing workflow. If you want to add something particular to your workflow, enclose the addition with something like
```
user = os.getenv("USER")
if user == 'ciperez':
```

Reply:
I just deleted line 36.